StreemNow/ViewController.swift

```diff
@@ -71,18 +51,26 @@ class ViewController: UIViewController {
                     let index = optionMenu.actions.index(of: alert)
                     let user = users[index!]
                     print("Calling user: \(user.id)")
-
+                    
                     Streem.sharedInstance.startRemoteStreem(asRole: .LOCAL_CUSTOMER, withRemoteUserId: user.id) { success in
                         if !success {
-                            let alert = UIAlertController(title: nil, message: "Unable to call \(user.name).", preferredStyle: .alert)
-                            alert.addAction(UIAlertAction(title: "OK", style: .default))
-                            self.present(alert, animated: true)
+                            self.presentAlert(message: "Unable to call \(user.name).")
+                            self.isStartingAStreem = false
                             return
                         }
+                        else {
+                            if #available(iOS 11.0, *) {
+                                let session = Streem.sharedInstance.getCurrentARSession()
+                                print("AR Session: \(String(describing: session))")
+                            }
```

Comment:
We can probably get rid of the useless `print`s we have around here

Reply:
👍